scipy/stats/stats.py

```diff
@@ -5466,16 +5477,30 @@ def combine_pvalues(pvalues, method='fisher', weights=None):
            for combining probabilities in meta-analysis." Journal of
            Evolutionary Biology 24, no. 8 (2011): 1836-1841.
     .. [5] https://en.wikipedia.org/wiki/Extensions_of_Fisher%27s_method
+    .. [6] Heard, N. and Rubin-Delanchey, P. "Choosing between methods of
+           combining p-values."  Biometrika 105.1 (2018): 239-246.
+    .. [7] George, E. O., and G. S. Mudholkar. "On the convolution of logistic
```

Comment:
Order initials consistent to the other references

Reply:
got it.